### ISSUE TITLE
Adventures blow up

### ISSUE THREAD

**Issue by eburggraaf** [kdechant/eamon#30]
Using htc u11 fully patched android O, chrome, talkback 7.10.2 and bluetooth keyboard.

In the middle of an adventure the adventure blows up with no warning or error message and dumps me to the go on an adventure page.  My brother who is using a galaxy s9 with no keyboard or accessibility features doesn't have this problem.

It happened to me once on my laptop windows 10 fully patched with nvda 2010.4.1 and google chrome.  On the laptop it's not less annoying, but it does seem to be less frequent.  I got through two or three adventures before one blew up.  On my phone I can barely attack the rats in the beginners cave before it's pooched.

Last night I tried playing on my phone in firefox.  It didn't blow up, but accessibility was horendous and I got fed up before finishing beginner's cave.  Another issue altogether.

I'm not even sure how to begin gathering information about this, but I'll provide anything that may help.

**Comment by kdechant** [comment]
That's strange. The only thing I can think of that would cause it to return to the adventure list would be if you had hit the back button on your browser.

I haven't actually tested Eamon very much on small screens and I'm sure there are improvements that could be made. I'll have to take a look, but it might take a few days before I can get to it.
